CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+# Changelot SessionInsights
+
+## Table of Contents
+
+- [v0.1.0 - Initial contribution](#v010---initial-contribution) (Fall25 public release)
```

Comment:
Please use r1.1 as headline and below, see https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/CHANGELOG_TEMPLATE.md and example in e.g. https://github.com/camaraproject/CallForwardingSignal/blob/main/CHANGELOG.md